.github/workflows/crowdin-download.yml

```diff
@@ -26,5 +26,5 @@ jobs:
           create_pull_request: true
           project_id: '492549'
           token: ${{ secrets.CROWDIN_PERSONAL_TOKEN }}
-          source: 'src/locales/en/messages.po'
-          translation: 'src/locales/%two_letters_code%/%original_file_name%'
+          source: 'src/locales/messages.pot'
+          translation: 'src/locales/%two_letters_code%/messages.po'
```

Comment:
`%original_file_name%` comes back as `messages.pot`, but we want `messages.po` - this ensures that.